<?xml version="1.0"?>
<launch>
  <!-- launch file parameters -->
  <arg name="debug"     default="false" />
  
  <arg if=    "$(arg debug)"  name="DEBUG" value="screen"/>
  <arg unless="$(arg debug)"  name="DEBUG" value="log"/>
  
  <!-- controllers config -->
  <rosparam file="$(find osr_control)/config/controllers.yaml" command="load" />
  <!-- load controllers -->
  <node name="robot_controllers" pkg="controller_manager" type="spawner" output="$(arg DEBUG)"
    args="joint_state_controller trajectory_controller gripper_controller"/>
  <!-- TF -->
  <node pkg="robot_state_publisher" type="state_publisher" name="state_publisher"> 
    <param name="publish_frequency"  type="double" value="50" />
  </node>
</launch>
